lib/t_web/views/feed_view.ex

```diff
@@ -4,25 +4,28 @@ defmodule TWeb.FeedView do
   alias T.Feeds.{FeedProfile}
 
   def render("feed_item.json", %{profile: profile, screen_width: screen_width}) do
-    %{"profile" => render_profile(profile, [:user_id, :name, :gender, :story], screen_width)}
+    profile =
+      render_profile(profile, [:user_id, :name, :gender, :story], screen_width, _env = :feed)
+
+    %{"profile" => profile}
   end
 
   def render("feed_profile.json", %{profile: profile, screen_width: screen_width}) do
-    render_profile(profile, [:user_id, :name, :gender, :story], screen_width)
+    render_profile(profile, [:user_id, :name, :gender, :story], screen_width, _env = :match)
   end
 
   def render("missed_call.json", %{profile: profile, call: call, screen_width: screen_width}) do
-    %{
-      "profile" => render_profile(profile, [:user_id, :name, :gender, :story], screen_width),
-      "call" => CallView.render("call.json", call: call)
-    }
+    profile =
```

Comment:
missed_call.json (& missed calls) can be deleted, version of the app with chat (93% adoption) do not use it at all; and for older version this won't be a breaking change as well